src/StackAdmin/Azs.Backup.Admin/Module/Azs.Backup.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsEncryptionKeyBase64.ps1

```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License. See License.txt in the project root for license
     Generate encryption key for infrastructure backups.
 
 #>
-function New-EncryptionKeyBase64
```

Comment:
Pls rename the example as well. Mention the parent cmdlet where this helper cmdlet would be used in the description

Reply:
Done.